src/screens/Home/HomeScreen/HomeScreen.tsx

```diff
@@ -48,12 +43,12 @@ export default function HomeScreen() {
           <Text style={styles.bannertext}>QUESTIONS</Text>
           <Image source={logo} style={styles.bannerlogo} />
         </View>
-        <FlatList
-          data={DATA}
+        <FlatList<IQuestion>
+          data={questions}
           renderItem={({ item }) => (
             <Item
               id={item.id}
-              title={item.title}
+              title={item.text}
               selected={selected === item.id}
               onSelect={() => setSelected(item.id)}
             />
```

Comment:
Read my overview comment for this PR, I reference this file

Reply:
Hi aliya,

Sorry, I was unclear on what this meant!

Thanks,

Brenna